sdk/src/main/java/com/oursky/authgear/latte/WebViewResult.kt

```diff
@@ -17,6 +18,11 @@ internal data class WebViewResult(private val finishUri: Uri) {
             val json = Base64.decode(base64Json, Base64.URL_SAFE).toString(Charsets.UTF_8)
             throw ServerException(JSONObject(json))
         }
+
+        if (!error.isNullOrBlank()) {
+            throw AuthgearException(error)
+        }
```

Comment:
This should be a OAuthError. Refer to iOS SDK on how to construct it.

Reply:
@louischan-oursky Updated, thanks!